File: PRDH/Domain/Models/CovidCaseSummary.cs
namespace PRDH.Domain.Models;

public class CovidCaseSummary
{
    public DateTime SampleCollectedDate { get; set; }
    public int QuantityOfCases { get; set; }
    public int QuantityOfCasesByMolecularTest { get; set; }
    public int QuantityOfCasesByAntigensTest { get; set; }
    public int QuantityOfCasesBySelfMadeAntigensTest { get; set; }
    public List<Case>? Cases { get; set; }
}

File: PRDH/ClientApp/src/app/covid-case/interfaces/covid-case-summary.ts
import {CovidCase} from "./covid-case";

export interface CovidCaseSummary {
  sampleCollectedDate: Date;
  quantityOfCases: number;
  quantityOfCasesByMolecularTest: number;
  quantityOfCasesByAntigensTest: number;
  quantityOfCasesBySelfMadeAntigensTest: number;
  cases: CovidCase[]
}
